packages/pds/src/api/com/atproto/admin/getInviteCodeUsage.ts

```diff
@@ -0,0 +1,74 @@
+import { Server } from '../../../../lexicon'
+import AppContext from '../../../../context'
+import { sql } from 'kysely'
+import { nullToZero } from '../../../../db/util'
+
+export default function (server: Server, ctx: AppContext) {
+  server.com.atproto.admin.getInviteCodeUsage({
+    auth: ctx.adminVerifier,
+    handler: async () => {
+      const ref = ctx.db.db.dynamic.ref
+      const res = await ctx.db.db
+        .with('use_count', (qb) =>
+          qb
+            .selectFrom('invite_code_use')
+            .groupBy('code')
+            .select(['code', sql<number>`count(*)`.as('uses')]),
```

Comment:
There's a `selectAll` helper in db utils that could be used here like `selectAll.as('uses')`.

Reply:
ah nice 👍